src/libcore/str/mod.rs

```diff
@@ -147,8 +147,8 @@ impl FromStr for bool {
     /// assert!(<bool as FromStr>::from_str("not even a boolean").is_err());
     /// ```
     ///
-    /// Note, in many cases, the StrExt::parse() which is based on
-    /// this FromStr::from_str() is more proper.
+    /// Note, in many cases, the `str::parse()` which is based on
+    /// this `FromStr::from_str()` is more proper.
```

Comment:
I know you didn't write this sentence, but while you're here, maybe update it? It sounds akward to me:

> Note, in many cases, the str::parse() function is more proper.